build.sbt

```diff
@@ -7,21 +7,21 @@ ThisBuild / organizationHomepage := Some(url("https://aiven.io/"))
 
 ThisBuild / resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 
-val akkaVersion                = "2.6.17"
+val akkaVersion                = "2.6.18"
 val akkaHttpVersion            = "10.2.7"
 val alpakkaKafkaVersion        = "2.1.1"
 val alpakkaVersion             = "3.0.4"
 val quillJdbcMonixVersion      = "3.7.2"
 val postgresqlJdbcVersion      = "42.2.24"
 val scalaLoggingVersion        = "3.9.4"
-val logbackClassicVersion      = "1.2.6"
+val logbackClassicVersion      = "1.2.10"
 val declineVersion             = "2.2.0"
-val pureConfigVersion          = "0.17.0"
+val pureConfigVersion          = "0.17.1"
 val scalaTestVersion           = "3.2.9"
 val scalaTestScalaCheckVersion = "3.2.9.0"
 val akkaStreamsJson            = "0.8.0"
-val diffxVersion               = "0.5.6"
-val testContainersVersion      = "0.39.8"
+val diffxVersion               = "0.7.0"
+val testContainersVersion      = "0.39.12"
 val testContainersJavaVersion  = "1.16.2"
```

Comment:
:D https://github.com/testcontainers/testcontainers-java/releases/tag/1.16.3

Reply:
Its not available on Maven yet so I will do another PR when its there